smsdk/client.py

```diff
@@ -403,9 +439,21 @@ def get_machine_schema(
 
         return frame
 
-    def get_fields_of_machine_type(
-        self, machine_type, types=[], show_hidden=False, **kwargs
-    ):
+    def get_fields_of_machine_type(self, *args, **kwargs):
```

Comment:
You can default the args to None instead of this

Reply:
This is the standard way of defining positional arguments. If there are no positional arguments passed, it will be come None. No need to set it explicitly.